### ISSUE TITLE
There is no image download menu in the piechart

### ISSUE THREAD

**Issue by wandresrh** [appreciated/apexcharts-flow#175]
There is no image download menu in the piechart, any solution?
I am using the example from [https://github.com/appreciated/apexcharts-flow/blob/master/src/test/java/com/github/appreciated/apexcharts/examples/pie/PieChartExample.java](url)
`withChart(ChartBuilder.get().withType(Type.PIE).build())
                .withLabels("Team A", "Team B", "Team C", "Team D", "Team E")
                .withLegend(LegendBuilder.get()
                        .withPosition(Position.RIGHT)
                        .build())
                .withSeries(44.0, 55.0, 13.0, 43.0, 22.0)
                .withResponsive(ResponsiveBuilder.get()
                        .withBreakpoint(480.0)
                        .withOptions(OptionsBuilder.get()
                                .withLegend(LegendBuilder.get()
                                        .withPosition(Position.BOTTOM)
                                        .build())
                                .build())
                        .build());`

![Captura](https://github.com/appreciated/apexcharts-flow/assets/59120513/58f9d2f3-a824-4ac5-a43f-f9322145c766)

thanks for your help

**Comment by wandresrh** [comment]
The solutions adding:
`.withToolbar(ToolbarBuilder.get().withShow(true).build())`

It would be nice if it were displayed automatically like the other types of graph.

**Comment by Loahrs** [comment]
I think this is the default behavior of the ApexCharts-JS library. Their pie charts don't seem to have a toolbar by default. 

Adding it here would make it kind of inconsistent i think.

**Comment by appreciated** [comment]
I agree with @Loahrs on this matter. The default behaviour is defined by the library.
